Source/SourceKittenFramework/File.swift

```diff
@@ -10,12 +10,18 @@ import Foundation
 import SwiftXPC
 import SWXMLHash
 
+public typealias Line = (index: Int, content: String)
```

Comment:
That's fair. I put it here just because this was the only place it was used. I'll move it